acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java

```diff
@@ -98,14 +99,50 @@ public ValidatorKeysApi getValidatorKeysApi() {
     return validatorKeysApi;
   }
 
-  public TekuValidatorNode withValidatorKeystores(ValidatorKeystores validatorKeystores) {
-    this.config.withValidatorKeys(
+  // will copy keystores onto the filesystem, so not read-only, except permissions are root access.
+  public TekuValidatorNode withReadOnlyKeystorePath(final ValidatorKeystores validatorKeystores) {
+    this.config.configMap.put("Xinterop-enabled", false);
+    this.config.configMap.put("validators-keystore-locking-enabled", false);
+    this.copyContentsToWorkingDirectory(validatorKeystores.getTarball());
+    this.config.configMap.put(
+        "validator-keys",
         WORKING_DIRECTORY
             + validatorKeystores.getKeysDirectoryName()
             + ":"
             + WORKING_DIRECTORY
             + validatorKeystores.getPasswordsDirectoryName());
-    this.copyContentsToWorkingDirectory(validatorKeystores.getTarball());
+    return this;
+  }
+
+  // will mount keystores folder as a read only filesystem
+  public TekuValidatorNode withReadOnlyKeystorePath(
```

Comment:
almost duplicated from TekuNode, we need to circle back at some point on the structure of these 2 classes